pkg/ottl/functions_test.go

```diff
@@ -1141,54 +1141,66 @@ func functionWithEnum(Enum) (ExprFunc[interface{}], error) {
 	}, nil
 }
 
-type reusableFunctionFactory[A any] struct {
-	name string
-	args A
-	fn   any
-}
+// func createFunction[K any](fCtx FunctionContext, oArgs Arguments) (ExprFunc[any], error) {
```

Comment:
Can these be removed?

Reply:
My bad, kept that for a second to copy some stuff. Removed.